src/index.html

```diff
@@ -7,8 +7,60 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Moyo header</h1>
+  <body class="body">
+    <header class="header">
+      <a class="logo" href="#home"><img src="./images/logo.png" alt="MOYO"></a>
+      <div></div>
+      <nav class="nav">
+        <ul class="nav__item">
+          <li class="nav__list">
+            <a class="is-active nav__link" href="#">Apple
+            </a>
+          </li>
+
+          <li class="nav__list">
+            <a class="nav__link" href="#">Samsung
+            </a>
+          </li>
+
+          <li class="nav__list">
+            <a class="nav__link" href="#">Smartphones
+            </a>
+          </li>
+
+          <li class="nav__list">
+            <a
+              class="nav__link"
+              href="#"
+              data-qa="hover">Laptops & Computers
+            </a>
```

Comment:
```suggestion
            <a
              class="nav__link"
              href="#"
              data-qa="hover"
            >
              Laptops & Computers
            </a>
```
![image](https://github.com/mate-academy/layout_moyo-header/assets/96409563/b488c867-5647-46ad-97ad-1093620f2c37)